web/main.js

```diff
@@ -194,6 +230,10 @@ window.addEventListener('keydown', function(e) {
       Main.clickNav();
       e.preventDefault();
     }
+    else if (e.key == 'Backspace' || e.keyCode == 8) {
```

Comment:
You can also pause the animation with `backspace` if you have a keyboard available (i.e. on desktop).